pkg/github/github.go

```diff
@@ -0,0 +1,26 @@
+package github
+
+import (
+	"context"
+	"os"
+
+	"github.com/google/go-github/v52/github"
```

Comment:
Hmm, we're already using `github.com/google/go-github/github` in get.go. Can these usages be consolidated? What's up with using `v52` here?

Reply:
Like mentioned in the general feed back, I've used the latest version for the new implementation.

Reply:
Solved by https://github.com/ddev/ddev/pull/4915